docs/types/reference-types.rst

```diff
@@ -470,6 +470,114 @@ Array Members
 
 .. index:: ! array;slice
 
+Dangling References to Storage Array Elements
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Maybe we should mention the "bugs" related to this in the blog post and then link to this section?

Reply:
We could, but I'd also say we don't have to :-).